src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/ManageRecallsApiExceptionHandlerTest.kt

```diff
@@ -40,6 +43,28 @@ class ManageRecallsApiExceptionHandlerTest {
     )
   }
 
+  @Test
+  fun `ClientTimeoutException should be translated to the corresponding status`() {
```

Comment:
"should be translated to the corresponding status" 😞   : nice to state actual intent in all these test names, IMO